index.html

```diff
@@ -167,6 +167,28 @@ <h1 class="about-title">About Myself</h1>
                 </ul>
             </section>
             </section>
+
+            <section class="last-section-mobile">
+                <h2>Contact me</h2>
+                    <p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it</p>
+                    <form action="#" method="post">
+                        <fieldset class="mobile-contact">
+                            <label for="username">
+                                <input type="text" class="input" required name="username" maxlength="30" placeholder="Username">
+                            </label>
+                            <label for="email">
+                                <input type="email" class="input" required name="email" placeholder="name@domain.com">
+                            </label>
+                            <label for="message">
+                                <textarea name="message" class="message-mobile" required maxlength="500" placeholder="Write your message here"></textarea>
+                            </label>
```

Comment:
Please stick to the design and add right amount of padding for each side of inputs. Here is the difference.
<img width="931" alt="Screenshot 2023-01-18 at 12 40 41 AM" src="https://user-images.githubusercontent.com/25887253/213020398-9173a3ad-cfc4-46b5-bb0d-c0ab0e581059.png">
<img width="598" alt="Screenshot 2023-01-18 at 12 41 14 AM" src="https://user-images.githubusercontent.com/25887253/213020401-46023b46-743d-458f-9e7a-e6b82f0d5acf.png">

Reply:
Thanks for the review @huseyinbiyik